tests/Functional/Driver/PDO/PgSQL/DBAL6044Test.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+namespace Doctrine\DBAL\Tests\Functional\Driver\PDO\PgSQL;
+
+use Doctrine\DBAL\Schema\Table;
+use Doctrine\DBAL\Tests\FunctionalTestCase;
+use Doctrine\DBAL\Tests\TestUtil;
+
+/** @requires extension pdo_pgsql */
+class DBAL6044Test extends FunctionalTestCase
+{
+    protected function setUp(): void
+    {
+        parent::setUp();
+
+        if (TestUtil::isDriverOneOf('pdo_pgsql')) {
+            return;
+        }
+
+        self::markTestSkipped('This test requires the pdo_pgsql driver.');
+    }
+
+    public function testUnloggedTables(): void
+    {
+        $unloggedTable = new Table('my_unlogged');
+        $unloggedTable->addOption('unlogged', true);
+        $unloggedTable->addColumn('foo', 'string');
+        $this->dropAndCreateTable($unloggedTable);
+
+        $loggedTable = new Table('my_logged');
+        $loggedTable->addColumn('foo', 'string');
+        $this->dropAndCreateTable($loggedTable);
+
+        $schemaManager = $this->connection->createSchemaManager();
+
+        $validationSchema = $schemaManager->introspectSchema();
+        $validationTable  = $validationSchema->getTable($unloggedTable->getName());
+        $this->assertEquals($unloggedTable->getName(), $validationTable->getName());
```

Comment:
Performing assertions on the table name does not make sense here. We already have tests that make sure a tables does not change its name unexpectedly.

Instead, we need to make sure, the new "unlogged" option is introspected correctly.